composer.json

```diff
@@ -30,7 +30,9 @@
             "@grunt:dependencies"
         ],
         "post-update-cmd": [
-            "@drupal:scaffold"
+            "@drupal:scaffold",
+            "@campaign:scaffold",
+            "@grunt:dependencies"
```

Comment:
@Saphyel why do we need to change this in this PR? Seems unrelated?

Reply:
it's related because now composer executes `update` (even if we run `composer install` see: 91d891b )